ReflectometryServer/server_status_manager.py

```diff
@@ -82,6 +82,7 @@ def _reset(self):
         self._status = STATUS.OKAY
         self._message = ""
         self._error_log = []
+        self._error_log_character_size = 0
```

Comment:
does this also need top be done on  `_clear_log` ? probably better to put the two operations in a separate function